pkgs/development/libraries/libraspberrypi/default.nix

```diff
@@ -1,25 +1,31 @@
-{ stdenv, cmake, fetchFromGitHub }:
+{ stdenv
```

Comment:
Sorry, this raced with https://github.com/NixOS/nixpkgs/pull/107689/files.

Can you incorporate the changes (`x86_64-linux` support) from there into here?